cmd/osquery-perf/agent.go

```diff
@@ -943,6 +976,103 @@ func (a *agent) runMacosMDMLoop() {
 				a.stats.IncrementMDMErrors()
 				break INNER_FOR_LOOP
 			}
+			if mdmCommandPayload != nil && mdmCommandPayload.Command.RequestType == "DeclarativeManagement" {
+				a.doDeclarativeManagement(mdmCommandPayload)
+			}
+		}
+	}
+}
+
+func (a *agent) doDeclarativeManagement(cmd *mdm.Command) {
+	// TODO: check tokens endpoint?
+	defer log.Printf("Exiting DeclarativeManagement for command %s", cmd.CommandUUID)
```

Comment:
if this branch is going to be merged in `main`, do we want this log for each host? I assumed it was going to be only for the DDM load test